spec/views/catalog/_view_type_group.html.erb_spec.rb

```diff
@@ -37,6 +37,21 @@
     expect(rendered).to have_selector('.view-type-c', :text => 'c')
   end
 
+  it "hides views that are set to not display" do
```

Comment:
I'd prefer that this is converted to a helper test as that's where all the logic resides.